### ISSUE TITLE
URL blocked by whitelist

### ISSUE THREAD

**Issue by albertogaona** [DavidFrahm/wakatime-mobile#3]
In Android I get 

`SystemWebViewClient﹕ URL blocked by whitelist: https://wakatime.com/api/v1/users/current/?api_key=XXXXXXXXXXXX`

**Comment by albertogaona** [comment]
Forgot to install cordova-plugin-whitelist plugin. Now it is working.

**Comment by DavidFrahm** [comment]
The whitelist is a new issue, caused by a new Cordova version than when I started the project.

I'm reopening this issue, so that the project gets updated for this.

**Comment by DavidFrahm** [comment]
The primary repo was moved to https://github.com/wakatime/wakatime-mobile.

I already added the whitelist stuff over there, in order to get a working build with the latest cordova platform libs.
